xclim/sdba/adjustment.py

```diff
@@ -134,10 +134,10 @@ def _harmonize_units(cls, *inputs, target: str | None = None):
         return (convert_units_to(inda, target) for inda in inputs), target
 
     @classmethod
-    def _train(ref, hist, **kwargs):
+    def _train(cls, hist, **kwargs):
```

Comment:
```suggestion
    def _train(cls, ref, hist, **kwargs):
```
My bad for this signature. The goal was to show what it should look like.